tests/address_test.go

```diff
@@ -1,132 +1,112 @@
 package easypost_test
 
 import (
+	"reflect"
+	"strings"
+
 	"github.com/EasyPost/easypost-go/v2"
 )
 
-func (c *ClientTests) TestAddressCreationVerification() {
+func (c *ClientTests) TestCreateAddress() {
 	client := c.TestClient()
-	assert, require := c.Assert(), c.Require()
-	// Create an address and then verify some fields to test whether it was
-	// created just fine.
-	address, err := client.CreateAddress(
-		&easypost.Address{
-			Company: "EasyPost",
-			Street1: "One Montgomery St",
-			Street2: "Ste 400",
-			City:    "San Francisco",
-			State:   "CA",
-			Zip:     "94104",
-			Phone:   "415-555-1212",
-		},
+	assert := c.Assert()
+
+	address, _ := client.CreateAddress(
+		c.fixture.BasicAddress(),
 		nil,
 	)
-	require.NoError(err)
-	address, err = client.VerifyAddress(address.ID)
-	require.NoError(err)
-	require.NotNil(address)
-	address, err = client.GetAddress(address.ID)
-	require.NoError(err)
-	require.NotNil(address)
-	assert.Equal("US", address.Country)
-	assert.Empty(address.Email)
-	assert.Empty(address.FederalTaxID)
-	assert.Equal("CA", address.State)
-	assert.Equal("94104-4533", address.Zip)
+
+	assert.Equal(reflect.TypeOf(&easypost.Address{}), reflect.TypeOf(address))
+	assert.True(strings.HasPrefix(address.ID, "adr_"))
+	assert.Equal("388 Townsend St", address.Street1)
 }
 
-func (c *ClientTests) TestAddressCreationWithVerify() {
+func (c *ClientTests) TestCreateVerifyStrictAddress() {
```

Comment:
Let's always have the names be the object name followed by the action, then the modifiers - this will help when searching through cassettes since the names reflect the test name and they aren't in separate folders.

```suggestion
func (c *ClientTests) TestAddressCreateVerifyStrict() {
```